sql/dolphinscheduler_h2.sql

```diff
@@ -328,6 +328,8 @@ CREATE TABLE t_ds_command
     worker_group              varchar(64),
     environment_code          bigint(20) DEFAULT '-1',
     dry_run                   int NULL DEFAULT 0,
+    process_instance_id       int(11) DEFAULT 0,
+    process_defintion_version int(11) DEFAULT 0,
```

Comment:
I found out we have typo here, we use `process_definition_version` in `Command.java`, but all database field use `process_defintion_version`

```suggestion
    process_defintion_version int(11) DEFAULT 0,
    process_definition_version int(11) DEFAULT 0,
```

Reply:
good suggestion. thx.